### PR TITLE
Update kerneldbgen to work with spks with many time intervals

### PR DESCRIPTION
**Pull request by krlberry**
Updated kerneldbgen to work with larger spks with many time intervals. 

## Description
Updated kerneldbgen to work with larger spks with many time intervals. 

## Related Issue
Needed to add Themis smithed kernels into ISIS as part of: #3363 

## Motivation and Context
See above. 

## How Has This Been Tested?
Ran kerneldbgen with the new smithed kernels. A new kernel database is generated. Additionally, spiceinit on an odyssey image with smithed=yes does result in it picking up the appropriate new kernel(s.)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/src/system/apps/kerneldbgen/SpiceDbGen.cpp [hunk 1]
```diff
@@ -224,10 +224,9 @@ PvlGroup SpiceDbGen::AddSelection(FileName fileIn, double startOffset, double en
 
       //find the correct coverage window
       if (currFile == "SPK") {
-        //  2000 is the max coverage window size for an SPK kernel
-        SPICEDOUBLE_CELL(cover, 2000);
+        SPICEDOUBLE_CELL(cover, 50000);
         ssize_c(0, &cover);
-        ssize_c(2000, &cover);
+        ssize_c(200000, &cover);
```
Comment (jessemapel) [inline isis/src/system/apps/kerneldbgen/SpiceDbGen.cpp:229]: Should this be the same number as in line 227?

Comment (krlberry) [reply isis/src/system/apps/kerneldbgen/SpiceDbGen.cpp:229]: Probably.

Comment (krlberry) [reply isis/src/system/apps/kerneldbgen/SpiceDbGen.cpp:229]: Thanks for catching that. Fixed!

**Review by jessemapel — APPROVED** [review]
This is good, but the PR template needs to be filled out with the linked issue, etc.

**Review by scsides — APPROVED** [review]

**Pull request merged**